Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -26,6 +27,8 @@
  */
 abstract class AbstractFilter implements FilterInterface
 {
+    use RequestTrait;
```

Comment:
Actually, no... I don't think traits are necessary in this case. Static methods on a util class would suffice.